### PR TITLE
refactor: cli flags

### PR DESCRIPTION
**Pull request by snitin315**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
No

### Motivation / Use-Case
Avoid additional negative flags.

### Breaking Changes
No

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]

File: bin/cli-flags.js [hunk 1]
```diff
@@ -88,7 +89,8 @@ module.exports = {
       type: Boolean,
```
Comment (anshumanv) [inline bin/cli-flags.js:89]: I think we no longer need these types, lets remove

Comment (snitin315) [reply bin/cli-flags.js:89]: I'm afraid we will lose compatibility with the webpack-cli version <= 4.5.0

Comment (anshumanv) [reply bin/cli-flags.js:89]: yep lets keep it

**Comment by alexander-akait** [comment]
Please don't merge PRs without my approve on webpack-dev-server, this packages is not stable, fir example, developer can set `bonjour: true` in config and use `--no-bonjour` to disable it, same for `open` and `client-progress`

**Comment by snitin315** [comment]
Oh yes, my bad. sorry about that, I will be more careful in the future. Should I revert one this then?

**Comment by alexander-akait** [comment]
Yes, feel free to send a PR

**Pull request merged**